app/scripts/metamask-controller.js

```diff
@@ -1334,13 +1334,9 @@ export default class MetamaskController extends EventEmitter {
         ],
       }),
       state: initState.SelectedNetworkController,
-      useRequestQueuePreference:
-        this.preferencesController.state.useRequestQueue,
+      useRequestQueuePreference: true,
       onPreferencesStateChange: (listener) => {
-        preferencesMessenger.subscribe(
-          'PreferencesController:stateChange',
-          listener,
-        );
+        listener({ useRequestQueue: true });
```

Comment:
can this hook just be a noop now instead? or do we have to immediately call the listener for some other reason?

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/29301/commits/c6dc756b754387a629469ad37a1d45dea7a8e275